test/tests/performance/pbs_qsub_performance.py

```diff
@@ -53,30 +52,27 @@ def setUp(self):
 
     def test_submit_job(self):
```

Comment:
I wouldn't call this test_submit_job.  You aren't just testing the submission of jobs.  You are testing the submission of jobs with and without large variables in the environment without the -V option.  I'd rename this to something like test_submit_large_env or something like that.